main/serve.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+let express = require('express');
```

Comment:
I don't think we're doing anything complex enough to warrant all these third party dependencies.

For serving let's use a [`Server`](https://github.com/foam-framework/foam2/blob/master/src/foam/net/node/Server.js) and [`StaticFileHandler`](https://github.com/foam-framework/foam2/blob/master/src/foam/net/node/StaticFileHandler.js) (and later add a [`RestDAOHandler`](https://github.com/foam-framework/foam2/blob/master/src/foam/net/node/RestDAOHandler.js)).

Also, we can parse a port number ourselves (and give reasonable error + help output). If we end up needing a bunch of params, we can revisit using `yargs` or simply model the params in FOAM.